exports/dataset.cpp

```diff
@@ -670,16 +710,21 @@ PYBIND11_MODULE(dataset, m) {
       .def(py::self += py::self, py::call_guard<py::gil_scoped_release>())
       .def(py::self -= py::self, py::call_guard<py::gil_scoped_release>())
       .def(py::self *= py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self /= py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self + py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self - py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self * py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self / py::self, py::call_guard<py::gil_scoped_release>())
       .def("__iadd__", [](VariableSlice &a, Variable &b) { return a += b; },
            py::is_operator())
       .def("__isub__", [](VariableSlice &a, Variable &b) { return a -= b; },
            py::is_operator())
       .def("__imul__", [](VariableSlice &a, Variable &b) { return a *= b; },
            py::is_operator())
+      .def("__len__", &VariableSlice::size)
```

Comment:
This is already defined above, with a different definition (the existing one matches the numpy definition, but I am not totally convinced it is the right way).

Reply:
I'll investigate but AFAIK, these overloads would not be generated by the shorthand above.

Reply:
What shorthand? Also `size` has no arguments, there is no overload?